src/mds/MDCache.cc

```diff
@@ -2000,14 +2000,19 @@ void MDCache::project_rstat_frag_to_inode(const nest_info_t& rstat,
 	(*_old_inodes)[last].first = first = ofirst;
       }
       pi = &(*_old_inodes)[last].inode;
-      pin->dirty_old_rstats.insert(last);
+      if (pi->btime >= rstat.rctime) {
+	pin->dirty_old_rstats.insert(last);
+      } else {
+	break;
+      }
```

Comment:
This indentation looks really weird.

Reply:
that's probably how the TAB character is emulated by the web browser code display component